huaweicloud/resource_huaweicloud_cce_cluster_v3.go

```diff
@@ -154,6 +154,12 @@ func ResourceCCEClusterV3() *schema.Resource {
 				ForceNew:     true,
 				ValidateFunc: validateIP,
 			},
+			"kubernetes_svcip_range": {
```

Comment:
suggest to named as `service_network_cidr`

Reply:
fixed